### DISCUSSION TITLE
2 step verification code don't work anymore because I don't have the old phone number anymore

### DISCUSSION THREAD

**Discussion by JarrodDeming** [community/community#134227] [Other Feature Feedback, Questions, & Ideas]
How do I bypass 2 step verification code when that is what they are asking me

**Comment by ghostinhershell** [comment]
Hey there! We know 2FA may be a new process for some folks and with that comes questions. We put together a [FAQ](https://github.com/orgs/community/discussions/68240) that will help answer your concerns above. If you still have more questions, feel free to ask them here.
